src/Illuminate/Routing/Router.php

```diff
@@ -531,6 +531,7 @@ protected function prependGroupNamespace($class)
         $group = end($this->groupStack);
 
         return isset($group['namespace']) && strpos($class, '\\') !== 0
+                && strpos($class, app()->getNamespace()) !== 0
```

Comment:
We can't call the `app()` function here. It exists only in the foundation package, which is not a dependency of the router.

Reply:
Thanks for reply, please see last comment. Didn't know about passing the array, which results in what was needed. Should I close the PR and make a new one for docs? I did manage to find details on the helpers page, but feel that many won't look there for answers.